### ISSUE TITLE
Please show details even when >100 sequences are uploaded

### ISSUE THREAD

**Issue by corneliusroemer** [yatisht/usher#298]
It looks like the new usher-sampled (on dev.usher.bio) can deal with well above 100 sequences 🎉 

Unfortunately the usefulness is reduced by not showing details which allows me to quickly open the trees.

Is there a fundamental reason you can't show details when there are >100 sequences?

<img width="1432" alt="image" src="https://user-images.githubusercontent.com/25161793/199059295-17769179-62ae-4259-b88e-310d1d606743.png">

**Comment by AngieHinrichs** [comment]
Yeah, you're right.  It makes the page output a lot bigger but doesn't really slow us down.  OK, I will make it just spew out the table and details for hundreds of sequences.  😄

**Comment by corneliusroemer** [comment]
Thank you! :)
